tsconfig.json

```diff
@@ -15,7 +15,7 @@
     // "sourceMap": true,                           /* Generates corresponding '.map' file. */
     // "outFile": "./",                             /* Concatenate and emit output to single file. */
      "outDir": "./dist",                              /* Redirect output structure to the directory. */
-     "rootDir": "./src",                             /* Specify the root directory of input files. Use to control the output directory structure with --outDir. */
+     "rootDir": "./",                             /* Specify the root directory of input files. Use to control the output directory structure with --outDir. */
```

Comment:
I'm not sure about how you want to handle that. The issue was that the `test` directory was not included in sources.
I can leave it this way or create separate tsconfig for test files @KrzysiekWyka

Reply:
That's OK, root dir can be `./` but this will/may change built files directory structure